app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -156,6 +156,7 @@
     private double latitude = 0;
     private double longitude = 0;
     private String source = "ip";
+    private String count;
```

Comment:
String??? count is integer, right ? Also, since count is related to rss, change variable name to rssCount or something similar.